module/platform/Webots/src/Webots.hpp

```diff
@@ -130,17 +131,14 @@ namespace module::platform {
         std::vector<std::pair<NUClear::clock::time_point, Eigen::Affine3d>> Hwps;
 
         struct CameraContext {
-            Webots& reactor;
             std::string name;
             uint32_t id;
             message::input::Image::Lens lens;
             // Homogenous transform from platform (p) to camera where platform is the rigid body the camera is attached
```

Comment:
Hpc implies camera to platform but comment says opposite

Reply:
And this is why you never put constants in comments when it can be avoided. Self documenting code good